app/app/static/custom/ourscript.js

```diff
@@ -43,18 +39,19 @@ $('#addToQueueForm').submit(function (e) {
         }
     })
 
+	const table = $(`#${team == 'STUDYSmarter' ? 'SS' : 'lib'}QueueTable tbody`);
 	table.append(
-		`<tr id="${id}" class="initialTime">
-		<td>${name}</td>
-		<td>${id}</td>
-		<td>${unit}</td>
-		<td class="text-right">${enquiry}</td>
-		<td class="text-right"><label id="minutes${id}">00</label><label id="colon">:</label><label id="seconds${id}">00</label></td>
-		<td class="td-actions text-right">
-		<button type="button" rel="tooltip" class="btn btn-success"><i class="material-icons">how_to_reg</i></button>
-		<button type="button" rel="tooltip" class="btn btn-danger" onclick="deleteRow(this)"><i class="material-icons">close</i></button></td>
-		</tr>`
-	)
+			`<tr id="${id}" class="initialTime">
+			<td>${name}</td>
+			<td>${id}</td>
+			<td>${unit}</td>
+			<td class="text-right">${enquiry}</td>
+			<td class="text-right"><label id="minutes${id}">00</label><label id="colon">:</label><label id="seconds${id}">00</label></td>
+			<td class="td-actions text-right">
+			<button type="button" rel="tooltip" class="btn btn-success" onclick="addSessionToTeam(team.value)(this)"><i class="material-icons">how_to_reg</i></button>
```

Comment:
`team.value?`

Reply:
maybe you want to "addSessionToTeam(${team})(this)"??

Reply:
when I used ${team.value}, it will say 'undefined' in the Team col of InSession queue, when I changed it to only 'team.value', it works fine

Reply:
okay sure as long as it works

Reply:
corrected in `dafd67`